pkg/authenticator/authenticator_test.go

```diff
@@ -27,26 +27,35 @@ func parseCert(filename string) (*x509.Certificate, error) {
 
 func TestAuthenticator(t *testing.T) {
 	Convey("IsCertExpired", t, func() {
-		Convey("Returns false if cert is not expired", func() {
+
+		Convey("Given a non-expired certificate to authenticate with Conjur", func() {
 			goodCert, err := parseCert("testdata/good_cert.crt")
-			So(err, ShouldBeNil)
 
-			authn := Authenticator{
-				PublicCert: goodCert,
-			}
+			Convey("Finishes without raising an error and returns certificate", func() {
+				So(err, ShouldBeNil)
```

Comment:
where do we verify that it returns a certificate?

Reply:
that is what parseCert returns, a cert and an error.